executable/tools-with-descriptions/README.md

```diff
@@ -0,0 +1,110 @@
+# Prescribed Tools with Descriptions
+
+This sample demonstrates how to use the `@tool` directive with prescribed operations that contains descriptions that can be used through to create MCP (Model Context Protocol) tools.
+
+## Overview
+
+The sample implements a mock weather service API with a simple operations, Weather forecast for a city
```

Comment:
"simple operations"

Remove "simple", and there's only a single operation right?